src/drivers/char/terminal/vtparse.c

```diff
@@ -46,6 +46,7 @@ static void do_action(struct vtparse *parser, vt_action_t action, char ch) {
 		break;
 
 	case VT_ACTION_COLLECT:
+		// Buffer overflow check may be needed
```

Comment:
I need these changes to make template compilable. I'm going to do rebase later when all other issues will be resolved.